source/js/foundation/template-js-handler/article-hero-video-handler.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * Adding play/pause functionality to the optional Article Page hero video section.
+ */
+
+ const articleHeroVideoHandler = () => {
+    let articlePageHero = document.querySelector(".article-hero");
+  
+    if (!articlePageHero) {
+      return;
+    }
+  
+    let video = articlePageHero.querySelector("video.hero-video");
+    let pauseButton = articlePageHero.querySelector(".btn-pause");
+    let playButton = articlePageHero.querySelector(".btn-play");
+  
+    function toggleVideo() {
+        video.paused ? video.play() : video.pause();     
+    }
+
+
+
+    if (video && pauseButton && playButton) {
+      const HIDE = `tw-hidden`;
+  
+      const showPauseButton = () => {
+        playButton.classList.add(HIDE);
+        pauseButton.classList.remove(HIDE);
+      };
+  
+      const showPlayButton = () => {
+        pauseButton.classList.add(HIDE);
+        playButton.classList.remove(HIDE);
+      };
+
+      video.addEventListener(`click`, () => {
+        toggleVideo();
+     });
+  
+      pauseButton.addEventListener(`click`, () => {
+        toggleVideo();
+      });
+  
+      playButton.addEventListener(`click`, () => {
+        toggleVideo();
+      });
+  
+      video.addEventListener(`pause`, () => {
+        showPlayButton();
+      });
+  
+      video.addEventListener(`playing`, () => {
+        showPauseButton();
+      });
+  
+      video.play();
+    }
+  };
+  
+  /**
+   * Bind handlers to article hero video
+   */
+  export default () => {
+    articleHeroVideoHandler();
+  };
```

Comment:
There is no reason to wrap this function in another function =)

```suggestion
  export default articleHeroVideoHandler
```

Although as the only thing in this file, it's even better to just start it as 

```js
export default function articleHeroVideoHandler() {
  ...
}
```